scripts/jenkins_build.sh

```diff
@@ -0,0 +1,28 @@
+# Build packages in Jenkins server.
+# The script expects a virtualenv created under ./env folder as prerequisite
+
+set -x  # do not echo command to prevent accidentally expose secrets
+
+. ./env/bin/activate
+
+echo 'Build Azure CLI and its command modules '
+
+pip install -r requirements.txt
```

Comment:
You shouldn't need to install all the requirements just to build.
`pip install --upgrade wheel` should be sufficient and this is required because you use `bdist_wheel` below.